tests/spread/integration/libnet1/task.yaml

```diff
@@ -0,0 +1,9 @@
+summary: Integration tests for libnet1
+
+execute: |
+  rootfs="$(install-slices libnet1_libs)"
+
+  apt install -y gcc
```

Comment:
a bit of a heavyweight approach to install gcc just for `-dumpmachine`. could make the CI slimmer with `arch=$(uname -m)-linux-gnu` ? `uname` is part of `coreutils` so i believe it ought to already be in the test env